docs/resources/volumegroup.md

```diff
@@ -26,6 +26,8 @@ description: |-
 
 VolumeGroup resource
 
+~> **Note:** Exactly one of `volume_ids` and `volume_names` should be provided.
```

Comment:
Exactly one of `volume_ids` and `volume_names`is required? Applicable for all instances.

Reply:
volume_ids/volume_names and protection_policy_id/protection_policy_name are not the required fields so, not mentioned as required in Note.